USAGE.md

```diff
@@ -0,0 +1,213 @@
+  },
+  "resource": {
+    "kind": "Deployment",
+    "namespace": "linkerd",
+    "name": "linkerd-controller"
+  },
+  "items": [
+    {
+      "kind": "TrafficMetrics",
+      "apiVersion": "metrics.smi-spec.io/v1alpha1",
+      "metadata": {
+        "name": "linkerd-controller",
+        "namespace": "linkerd",
+        "selfLink": "/apis/metrics.smi-spec.io/v1alpha1/namespaces/linkerd/deployments/linkerd-controller/edges",
+        "creationTimestamp": "2019-06-17T14:51:57Z"
+      },
+      "timestamp": "2019-06-17T14:51:57Z",
+      "window": "30s",
+      "resource": {
+        "kind": "Deployment",
+        "namespace": "linkerd",
+        "name": "linkerd-controller"
+      },
+      "edge": {
+        "direction": "from",
+        "resource": {
+          "kind": "Deployment",
+          "namespace": "linkerd",
+          "name": "linkerd-web"
+        }
+      },
+      "metrics": [
+        {
+          "name": "p99_response_latency",
+          "unit": "ms",
+          "value": "294"
+        },
+        {
+          "name": "p90_response_latency",
+          "unit": "ms",
+          "value": "240"
+        },
+        {
+          "name": "p50_response_latency",
+          "unit": "ms",
+          "value": "150"
+        },
+        {
+          "name": "success_count",
+          "value": "28580m"
+        },
+        {
+          "name": "failure_count",
+          "value": "0"
+        }
+      ]
+    },
+    {
+      "kind": "TrafficMetrics",
+      "apiVersion": "metrics.smi-spec.io/v1alpha1",
+      "metadata": {
+        "name": "linkerd-controller",
+        "namespace": "linkerd",
+        "selfLink": "/apis/metrics.smi-spec.io/v1alpha1/namespaces/linkerd/deployments/linkerd-controller/edges",
+        "creationTimestamp": "2019-06-17T14:51:58Z"
+      },
+      "timestamp": "2019-06-17T14:51:57Z",
+      "window": "30s",
+      "resource": {
+        "kind": "Deployment",
+        "namespace": "linkerd",
+        "name": "linkerd-controller"
+      },
+      "edge": {
+        "direction": "to",
+        "resource": {
+          "kind": "Deployment",
+          "namespace": "linkerd",
+          "name": "linkerd-prometheus"
+        }
+      },
+      "metrics": [
+        {
+          "name": "p99_response_latency",
+          "unit": "ms",
+          "value": "368"
+        },
+        {
+          "name": "p90_response_latency",
+          "unit": "ms",
+          "value": "247199m"
+        },
+        {
+          "name": "p50_response_latency",
+          "unit": "ms",
+          "value": "120731m"
+        },
+        {
+          "name": "success_count",
+          "value": "1008100m"
+        },
+        {
+          "name": "failure_count",
+          "value": "0"
+        }
+      ]
+    }
+  ]
+}
+```
+
+The support for istio and consul connect is being worked up on right now and the API and responses will have the same structure unless there are no changes to the spec.
```

Comment:
This should definitely be its own section in the README and not just at the bottom here =)